### ISSUE TITLE
🖼️ Design Wallpapers and Add Them to the Project

### ISSUE THREAD

**Issue by Parnab03** [Parnab03/WallGodds#8]
**Description:** Create minimalistic wallpapers aligned with the project theme. Add them to the `wallpapers` folder and write a description in `wallpapers.md`.
**Deliverable:** The wallpapers with proper naming `(e.g., yourname-pullrequestnumber.jpg)` and documented descriptions.

**Comment by dinesh-2047** [comment]
assign this issue to me. I want to add supercar wallpaper for mobiles .

**Comment by Parnab03** [comment]
> Hi,[@Parnab03](https://github.com/Parnab03) Assign this issue to me. I would like to add a supercar wallpaper for mobile devices.

I am assigning the issue to you, but the wallpaper should be either created by you or an image you have clicked yourself. If we find that you have used an image copied from the internet and recognize it, you will be permanently banned from the community.

**Comment by dinesh-2047** [comment]
Can we add AI-generated wallpapers?

**Comment by Parnab03** [comment]
> Can we add AI-generated wallpapers?

Yes, but it needs to be clear and cost-effective, not cheap. Before raising a PR, please share your work with us in the Discord wallpapers discussion channel.

**Comment by dinesh-2047** [comment]
ok

**Comment by imsudiptaa** [comment]
Hello @Parnab03 , I want to be assigned in this task.
I would like to add my personally clicked images.

**Comment by Parnab03** [comment]
> Hello [@Parnab03](https://github.com/Parnab03) , I want to be assigned in this task. I would like to add my personally clicked images.

yhaa sure.

**Comment by dinesh-2047** [comment]
hello @Parnab03 i would like to add more wallpapers .

**Comment by Parnab03** [comment]
@dinesh-2047 sure.

**Comment by Arushi-6278** [comment]
I'd like to add some images clicked by myself.

**Comment by SoumikGhosh9** [comment]
@Harshhdhankhar 
**Thanks for choosing WallGodds!**

Before you begin working on the issue, please take a moment to go through our contribution guidelines:

1. **Join our WallGodds Discord Server**:
   👉 [Join here](https://discord.gg/kTQ5KWANp8)
   It's important to stay connected. Communicating only through GitHub issue comments can be difficult and may cause delays. Please use the discussion channels on Discord to share updates, ask questions, and submit your work.
   Once your task is complete, **submit a short demo video** in the discussion channel. A maintainer will review it, and after approval, you're welcome to raise a Pull Request (PR).
   ⚠️ *Note: If you raise a PR without prior communication or approval on Discord, it will be disqualified.*

2. **Read the Contribution Guidelines**:
   Please read our [Contribution Guidelines](https://github.com/Parnab03/WallGodds/discussions/97) carefully to understand our workflow and expectations.

Now that you're aware of the process and agree to follow the rules, I’m assigning the issue to you. Looking forward to your contribution and seeing a PR from you soon!

**Comment by SoumikGhosh9** [comment]
Hi @sharanya2504 
As per our contribution [Guidelines](https://github.com/Parnab03/WallGodds/discussions/97), issues have a limited active time window after assignment.

Unfortunately, you haven't provided any updates or responses regarding your assigned issue within this timeframe. ⏳

👉 This is our final reminder to share your progress or updates in Discord, or let us know if you're facing any challenges — in that case, we will happily extend your time and help you out.

However, if we do not receive a response within the next 24 hours, we will have to disassign you from this issue to keep the project moving forward.

Thanks for understanding and being a part of the community! 💙

**Comment by SoumikGhosh9** [comment]
@sharanya2504 
You haven't discussed your work in the Discord channel, as outlined in the guidelines, and have directly raised a PR. As a result, your PR will be closed. Please initiate a discussion regarding your work in the Discord channel, and once it receives approval, you may proceed with raising a new PR.

**Comment by SoumikGhosh9** [comment]
Hello @sharanya2504 , since it's been over 24 hours and we've not received any response from you, we are now proceeding to unassign you from this issue in order to keep the project progressing smoothly.

We appreciate your initial interest, and you're always welcome to contribute again in the future. If you faced any difficulties or need assistance, feel free to reach out on [Discord](https://discord.gg/kTQ5KWANp8) — we’re here to help! 💬

Thanks you!
